demo/views/demo-all.dust

```diff
@@ -34,14 +35,11 @@
   </div>
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
Hmm… Your code seems to be working with:

```html
<!--<script src="/carbon-components.min.js"></script>-->
<script src="/demo.js"></script>
```

So I’m curious what was the issue you hit?